packages/fx-core/src/plugins/resource/localdebug/tasks.ts

```diff
@@ -27,6 +33,7 @@ export function generateTasks(includeFrontend: boolean, includeBackend: boolean,
                 label: "Pre Debug Check",
                 dependsOn: [
                     "dependency check",
+                    "detect ports in use",
```

Comment:
Move into predebugcheck, not add new task...

Reply:
Updated. Adding new task makes it more complex...